ngraph/test/constant_folding.cpp

```diff
@@ -2421,7 +2421,7 @@ TEST(constant_folding, constant_v1_one_hot)
 
 TEST(constant_folding, constant_v1_one_hot_negative_axes)
```

Comment:
This test name is `constant_v1_one_hot_negative_axes` which suggest me that test was intended for negative axes testing, which suggest they should be supported.  I see that you change spec and said explicitly that negative indexes are not supported,  you consulted this change with someone right ? :) If so, then I think we can remove this test at all, or at least rename it - now it is confusing.

Reply:
@jdanieck, you are right, the behavior for negative indices is undefined according to the new version of specs (please, see the [discussion](https://github.com/openvinotoolkit/openvino/pull/4243#issuecomment-778197662) for details). However, negative axis values are still allowed and supported (please, see the [specs](https://github.com/openvinotoolkit/openvino/blame/c001e54403c3ead20f6fc4cf96b290af00895002/docs/ops/sequence/OneHot_1.md#L25-L29) and [implementation](https://github.com/openvinotoolkit/openvino/blob/3a5eded6af710402ef8de2482af0f03b65313bd5/ngraph/core/src/op/one_hot.cpp#L84-L85)).